features/step_definitions/dashboard_steps.rb

```diff
@@ -0,0 +1,56 @@
+Given('I am on the dashboard') do
+  steps %(
+    Given I am on the sign in page
+    And there is a building in the system
+    And I log in
+  )
+end
+
+Given('I am on the sign in page') do
+  visit "/admin"
+end
+
+And('there is a building in the system') do
```

Comment:
We have a very similar step that could probably be extracted and reused: https://github.com/unboxed/southwark-bsp/blob/fix/new-sections/features/step_definitions/survey_steps.rb#L11

Reply:
I didn't spot this! Good suggestion -> less code :)